src/app/content/components/Attribution.tsx

```diff
@@ -96,6 +96,44 @@ const AttributionDetails = styled(Details)`
 
   ${disablePrint}
 `;
+// date is initialized as UTC, conversion to local time can change the date.
+// this compensates
+function compensateForUTC(date: Date): void {
+  date.setMinutes(date.getMinutes() + date.getTimezoneOffset());
+}
+
+function getPublishDate(book: Book): Date | null {
+  if (!('publish_date' in book)) {
+    return null;
+  }
+  const date = new Date(book.publish_date);
+
+  compensateForUTC(date);
+  return date;
+}
+
+function getAuthors(book: Book) {
+  if (!('authors' in book)) {
+    return [];
+  }
+  const seniorAuthors = book.authors.filter((author) => author.value.senior_author);
+
+  return seniorAuthors.length > 0
+    ? seniorAuthors
+    : book.authors.slice(0, 2);
```

Comment:
I know this is code that was already there so normally I wouldn't complain... too much. Except that now we're using it when making actual citations...

According to the MLA: `If a source has two authors, name both authors in your MLA in-text citation and Works Cited entry. If there are three or more authors, name only the first author, followed by et al.`

But we're not necessarily using MLA... who is responsible for our citation format anyway?

I won't hold back the PR because of this though... we can always make another card to fix.